Hq final

Shp0i opened the pull request:
Labeled and duplicated base whitebox, ready for second pass/revising

Shp0i committed:
WIP v1

Added placeholder labels for achievement items, upgrades, and general building locations.

Shp0i committed:
WIP v2

Duplicated the base whitebox for merging

GildedToxin reviewed (approved):
I can't open and view map but files look from what I can tell on git mobile

Shp0i committed:
WIP v3

Tweaked mountains, landscapes, and props; added various new props; tweaked labels
